etbrasil/geesebal/tools.py

```diff
@@ -1,691 +1,779 @@
-#----------------------------------------------------------------------------------------#
-#---------------------------------------//GEESEBAL//-------------------------------------#
-#GEESEBAL - GOOGLE EARTH ENGINE APP FOR SURFACE ENERGY BALANCE ALGORITHM FOR LAND (SEBAL)
-#CREATE BY: LEONARDO LAIPELT, RAFAEL KAYSER, ANDERSON RUHOFF AND AYAN FLEISCHMANN
-#PROJECT - ET BRASIL https://etbrasil.org/
-#LAB - HIDROLOGIA DE GRANDE ESCALA [HGE] website: https://www.ufrgs.br/hge/author/hge/
-#UNIVERSITY - UNIVERSIDADE FEDERAL DO RIO GRANDE DO SUL - UFRGS
-#RIO GRANDE DO SUL, BRAZIL
-
-#DOI
-#VERSION 0.1.1
-#CONTACT US: leonardo.laipelt@ufrgs.br
-
-#----------------------------------------------------------------------------------------#
+# ----------------------------------------------------------------------------------------#

```

Comment:
Oh you are right, it has been formatted. I'll try to explain it as best as possible here

Basically in this file I have eliminated everything that has to do with the brightness temperature since in the new collection they provide us directly the LST

Line 74: LST calculation  is no longer necessary 
Line 87: brt_r is no longer necessary
Line 161: The same
Line 247: LStemp is no longer necessary